backend/prompt_studio/prompt_studio_index_manager/models.py

```diff
@@ -78,3 +90,39 @@ class Meta:
                 name="unique_document_manager_profile_manager",
             ),
         ]
+
+
+# Function will be executed every time an instance of IndexManager is deleted.
+@receiver(post_delete, sender=IndexManager)
+def perform_vector_db_cleanup(sender, instance, **kwargs):
+    """Signal to perform vector db cleanup."""
+    logger.info("Performing vector db cleanup")
+    # Get the index_ids_history to clean up from the vector db
+    index_ids_history = json.loads(instance.index_ids_history)
+    embedding_instance_id = str(instance.profile_manager.embedding_model.id)
+    vector_db_instance_id = str(instance.profile_manager.vector_store.id)
+    # Generate a run_id
+    run_id = CommonUtils.get_uuid()
```

Comment:
@athul-rs what is the purpose of this??

Reply:
@athul-rs I updated the SDK to make embedding and vectordb to work without `run_id` in `0.29.0`

Reply:
@Deepak-Kesavan Thanks. Will make this update.